src/analysis/openstreetmap/qgsosmdownload.h

```diff
@@ -83,8 +84,11 @@ class ANALYSIS_EXPORT QgsOSMDownload : public QObject
     bool isFinished() const;
 
   signals:
-    void finished(); //!< Emitted when the network reply has finished (with success or with an error)
-    void downloadProgress( qint64, qint64 ); //!< Normally the total length is not known (until we reach end)
+    //!< Emitted when the network reply has finished (with success or with an error)
+    void finished();
+
+    //!< Normally the total length is not known (until we reach end)
+    void downloadProgress( qint64, qint64 );
```

Comment:
strange, this seems to not recognize the documentation. Can you try to remove the `<` after `!` ?